.pre-commit-config.yaml

```diff
@@ -25,6 +25,9 @@ repos:
     hooks:
     -   id: ruff
         args: [--exit-non-zero-on-fix]
+    -   id: ruff
+        name: ruff's autotyping/flake8-annotations autofixes
+        args: [--select, ANN, --fix-only, --exit-non-zero-on-fix]
```

Comment:
Could we just include `ANN` as a rule in pyproject.toml?

Reply:
I wish, but there are way too many violations (ANN wants everything to be annotated) and only some of them have in some cases an autofix. I would like only to use the autofixes. If there is a better way to achieve that, I'm all for that - but I didn't find one.

Reply:
Hmm since it appears `ANN204` is the only auto-fix rule so far, could we just include that code?

Reply:
Let me put this back to draft and wait until #54271 is merged

Reply:
There are 191 methods without return annotations.

Reply:
I currently do not have time to add 191 return annotations but I will start working on it in the future.

ruff might have autofixes for ANN001 in the future https://github.com/astral-sh/ruff/issues/1640